backend/api/views/main.py

```diff
@@ -38,32 +39,28 @@ def cal_apr():
 """
     assume a query_type argument to specify what to get
 """
-@app.route(INFO_INPUT_URL)
+@app.route(GET_VERSION_NUM)
 def get_info():
```

Comment:
maybe if you can rename this to get_version so that its easier to understand from a reading perspect